source/features/profile-gists-link.tsx

```diff
@@ -44,6 +45,5 @@ void features.add(__filebasename, {
 	include: [
 		pageDetect.isUserProfile,
 	],
-	awaitDomReady: false,
```

Comment:
Why was this removed?

Reply:
https://github.com/sindresorhus/refined-github/issues/4751#issuecomment-912606783

Reply:
So then the elementReady should be removed